{"name": "Ошибки авторизации с неверным логином/паролем", "status": "passed", "steps": [{"name": "Регистрация нового курьера и возвращение логина и пароля", "status": "passed", "start": 1734291878876, "stop": 1734291879056}, {"name": "Авторизация курьера", "status": "passed", "parameters": [{"name": "login", "value": "'wrong_login'"}, {"name": "password", "value": "'test_password'"}], "start": 1734291879056, "stop": 1734291879212}], "parameters": [{"name": "wrong_login", "value": "'wrong_login'"}, {"name": "wrong_password", "value": "'test_password'"}], "start": 1734291878875, "stop": 1734291879213, "uuid": "14f44333-9a0d-423c-ac52-b1e97422c558", "historyId": "594639d6ca6722f14ead7272d923eabe", "testCaseId": "1f2bce4b16f18147086031658deae890", "fullName": "tests.test_login_courier.TestCourierAuthorization#test_authentication_errors", "labels": [{"name": "suite", "value": "Тесты на проверку логина курьера"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestCourierAuthorization"}, {"name": "host", "value": "LAPTOP-PCJM6AA3"}, {"name": "thread", "value": "19776-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}